corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -90,6 +90,7 @@ def ensure_grant(self, grantee_slug, priv_slug, dry_run=False):
             grantee = Role.objects.get(slug=grantee_slug)
             priv = Role.objects.get(slug=priv_slug)
 
+            Role.get_cache().clear()
             if grantee.has_privilege(priv):
```

Comment:
Took me forever to track this down. I struggled with HQ tests because they're so slow and the console output is overwhelming and mostly useless. I was hoping to find a way to log SQL queries while running tests, but that proved fruitless until I went directly to postgres and enabled statement logging.